packages/radix-ui-themes/src/props/height.props.ts

```diff
@@ -1,20 +1,53 @@
 import { GetPropDefTypes, PropDef } from './prop-def.js';
 
 const heightPropDefs = {
+  /**
```

Comment:
were those missed in the other PR? I can swear I'd seen the width/height stuff in that PR lol